README.md

```diff
@@ -0,0 +1,91 @@
+
+# Unified Post-Processing (UPP)
+
+The Unified Post Processing (UPP) package provides tools for post
+processing for NCEP models. For historical reasons, the UPP repository
+is called EMC_post.
+
+This is part of the [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS)
+project. It also serves as a standalone package that is distributed to
+the community. It is built outside of nceplibs for some UFS
+applications (e.g. SRW v1.0.0).
+
+## Authors
+
+NCEP/EMC Developers
+
+Code Manager: Wen Meng, Huiya Chuang, Kate Fossell
+
+## Prerequisites
+
+This package requires the following NCEPLIBS packages:
+
+- [NCEPLIBS-g2](https://github.com/NOAA-EMC/NCEPLIBS-g2)
+- [NCEPLIBS-g2tmpl](https://github.com/NOAA-EMC/NCEPLIBS-g2tmpl)
+- [NCEPLIBS-sp](https://github.com/NOAA-EMC/NCEPLIBS-sp)
+- [NCEPLIBS-ip](https://github.com/NOAA-EMC/NCEPLIBS-ip)
+- [NCEPLIBS-bacio](https://github.com/NOAA-EMC/NCEPLIBS-bacio)
+- [NCEPLIBS-w3emc](https://github.com/NOAA-EMC/NCEPLIBS-w3emc)
```

Comment:
A new line might be needed:
[NCEPLIBS-w3nco](https://github.com/NOAA-EMC/NCEPLIBS-w3nco)

Reply:
Added.